variants/backend-base/config/environments/production.rb

```diff
@@ -42,13 +42,9 @@
   RUBY
 end
 
-gsub_file "config/environments/production.rb",
-          "config.log_level = :info",
-          'config.log_level = ENV.fetch("LOG_LEVEL", "info").to_sym'
```

Comment:
note: Rails 7.1 now provides this out of the box (it doesn't have the `to_sym` call, but I assume that it's fine since Rails is the one doing the generating 😅 )